test/runtests.jl

```diff
@@ -121,15 +121,30 @@ include("geojson_samples.jl")
         @test iterate(p, 2) === (2.2, 3)
         @test iterate(p, 3) === nothing
 
+        # Mixed name vector
+        f2 = GeoJSON.Feature(p; properties = (a = 1, geometry = "g", b = 2, c = 3))
+        GeoJSON.FeatureCollection((type = "FeatureCollection", features = [f, f2]))
+    end
+
+    @testset "Tables" begin
+        p = GeoJSON.Point(coordinates = [1.1, 2.2])
+        f = GeoJSON.Feature(p; properties = (a = 1, geometry = "g", b = 2))
+        features = [f]
         # other constructors
         @test DataFrame([GeoJSON.Feature(geometry = p, properties = (a = 1, geometry = "g", b = 2))]) ==
               DataFrame([GeoJSON.Feature((geometry = p, properties = (a = 1, geometry = "g", b = 2)))]) ==
-              DataFrame(GeoJSON.FeatureCollection((type="FeatureCollection", features=[f]))) ==
+              DataFrame(GeoJSON.FeatureCollection((; type="FeatureCollection", features))) ==
               DataFrame(GeoJSON.FeatureCollection(; features))
 
-        # Mixed name vector
-        f2 = GeoJSON.Feature(p; properties = (a = 1, geometry = "g", b = 2, c = 3))
-        GeoJSON.FeatureCollection((type = "FeatureCollection", features = [f, f2]))
+        # Round trip DataFrame -> FeatureCollection -> DataFrame 
+        features = [GeoJSON.Feature(geometry = p, properties = (a = 1, b = 2)), GeoJSON.Feature(geometry = p, properties = (a = 5, b = 10))] 
+        df = DataFrame(GeoJSON.FeatureCollection(features))
+        @test df == DataFrame(GeoJSON.FeatureCollection(df))
+
+        df_custom_col = DataFrame(:points => [p, p], :x => [1, 2])
```

Comment:
Does it work with other non-GeoJSON.jl geometry types as well?

Reply:
Good question! I actually forgot about that. So we need the other `convert` PR merged first.

I'll get that passing and come back to this. It's also a good example of how the `traittype` method will be useful from within a package as well.